doc/poo.md

```diff
@@ -380,6 +388,15 @@ Each entry in `slot-definitions` specifies how to compute a given named slot:
      ```
      (slot-name (inherited-computation) form)
      ```
+     
+     A simple example, where we have a computation `f` which evaluates to 1,
+     a special form `id` which expands any expression to itself.
```

Comment:
`special form` in Lisp means a macro or primitive that doesn't behave like a function and `expands` refer to what macros do. This is a function that returns its unmodified argument.